docs/src/Box.doc.js

```diff
@@ -638,23 +757,51 @@ function ButtonFlyoutExample() {
     </React.Fragment>
   );
 }`}
-  />,
-);
-
-card(
-  <Example
-    description={`
-It's possible to use box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.
-  `}
-    id="zindex"
-    name="ZIndex"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`It's possible to use Box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.`}
+      title="Z-Index"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 function Example() {
   const HEADER_ZINDEX = new FixedZIndex(100);
   const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
-  return <Box color="blue" width={60} height={60} zIndex={zIndex} />
+  return <Box color="midnight" width={60} height={60} zIndex={zIndex} />
```

Comment:
I think this would be better demonstrated with two (or more) Boxes with various z-indices? A single Box doesn't convey the layering aspect

Reply:
Done :D